input/pagecontent/mcsd-hpd-mapping.md

```diff
@@ -0,0 +1,235 @@
+| `deceased`                             | `false`                    | `#deceased`                                                      |
+
+NB: HPD `HCRegulatedOrganization.hpdProviderStatus` has only two possible values &mdash; `active` and `inactive` &mdash;
+and thus does not need any additional definitions to be mapped to FHIR `Organization.active` of type `boolean`.
+
+## Mapping of organization names
+
+HPD `HCRegulatedOrganization` defines two types of organization names &mdash; legal names (`hcRegisteredName`) and
+other names (`O`), both with cardinalities 1..n. In FHIR `Organization` resource, their counterparts are
+`name` and `alias`, respectively. But `name` has the cardinality 0..1. Additional legal organization names in FHIR
+can be placed into `Organization.name.extension[registeredNames].valueString`.
+
+## Mapping of HPD Relationship objects
+
+An HPD `Relationship` object represents a group of individual and organization healthcare providers ("members")
+belonging to a parent organization ("owner"). The list of members can be empty. Each HPD `Relationship` has its own
+identity (`cn`). An organization can be owner of multiple relationships, but member of only a single one. An individual
+provider can be member of multiple relationships.
+
+There is no direct mapping of HPD `Relationship` to FHIR mCSD resources, because in mCSD, the affiliation with a parent
+`Organization` is expressed not groupwise, but individually for each individual member (`PractitionerRole.organization`)
+and organization member (`Organization.partOf`).
+
+To allow a mapping between these very different models, the following approach is defined:
+
+* `Organization.extension[ownerOf]` (0..*) of the type `HpdUidIdentifier` contains a list of CNs of HPD `Relationship`
+  objects where this `Organization` is the owner. This is necessary in particular to allow relationships without
+  members.
+* `Organization.partOf.extension[memberOf]` (1..1) of the same type contains the CN of the HPD `Relationship`
+  to which this link between two organizations belongs.
+* `PractitionerRole.identifier` has a slice "memberOf" (1..1) of the same type
+  which contains the CN of the HPD `Relationship` to which the link between `PractitionerRole.practitioner`
+  and `PractitionerRole.organization` belongs.
+
+The following constraints apply:
+
+* If any "memberOf" CNs of any two FHIR resources (`Organization` or `PractitionerRole`) are equal,
+  then these resources shall reference the same parent `Organization`.
+* No two `Organization` resources shall have the same CN in their "ownerOf" lists,
+  i.e. no two organizations shall own the same relationship.
+* Each "memberOf" CN shall occur in the "ownerOf" list of exactly one `Organization`.
+
+Implementers of mCSD Directory actors shall assure that these constraints are fulfilled, and that the
```

Comment:
I think this chapter description needs to be rewritten, the intention of CH mCSD is the representation via Organization as a representation of teams, suggesting a parallel requirement of adding extensions seems like duplication and not useful (why do we have to define those extensions? the client is not interested in them and would not use it in anyway). Or do you  see that different?